src/data/SessionTracker.cpp

```diff
@@ -62,7 +62,10 @@ void SessionTracker::LoadSessions()
 
             const auto nSessionLength = strtoul(&sLine[nIndex2], nullptr, 10);
 
-            auto md5 = RAGenerateMD5(reinterpret_cast<const unsigned char*>(sLine.c_str()), nIndex + 1);
+            
+#pragma warning(suppress: 26490)
+            GSL_SUPPRESS_TYPE1 auto md5 =
+                RAGenerateMD5(gsl::make_span(reinterpret_cast<const unsigned char*>(sLine.c_str()), nIndex + 1));
```

Comment:
There is a function in gsl called `as_bytes`, but that's for spans of `const std::byte*`. Plus it had to do `reinterpret_cast` anyway so it didn't seem worth the hassle.